histomicstk/features/ComputeMorphometryFeatures.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import pandas as pd
+from skimage.measure import regionprops
+
+
+def ComputeMorphometryFeatures(im_label):
+    """
+    Calculates morphometry features for each object
+
+    Parameters
+    ----------
+    im_label : array_like
+        A labeled mask image wherein intensity of a pixel is the ID of the
+        object it belongs to. Non-zero values are considered to be foreground
+        objects.
+
+    rprops[i] : property
+        A rprops[i] property from regioprops.
+
+    Returns
+    -------
+    fdata: pandas.DataFrame
+        A pandas dataframe containing the morphometry features for each
+        object/label listed below.
+
+    Notes
+    -----
+
+    List of morphometry features computed by this function:
+
+    Area : int
+        Number of pixels the object occupies.
+
+    Circularity: float
+        A measure of how similar the shape of an object is to the circle
+
+    Eccentricity : float
+        A measure of aspect ratio computed to be the eccentricity of the
+        ellipse that has the same second-moments as the region. Eccentricity
+        of an ellipse is the ratio of the focal distance (distance between
+        focal points) over the major axis length. The value is in the
+        interval [0, 1). When it is 0, the ellipse becomes a circle.
+
+    EquivalentDiameter : float
+        The diameter of a circle with the same area as the object.
+
+    Extent : float
+        Ratio of area of the object to its axis-aligned bounding box.
```

Comment:
@cooperlab I think this needs to be compute relative to the tightest enclosing bounding box for most invariance.